public/components/datasources/components/manage/associated_objects/utils/associated_objects_tab_utils.tsx

```diff
@@ -228,7 +241,7 @@ export const ASSC_OBJ_TABLE_SEARCH_HINT =
 
 export const ASSC_OBJ_PANEL_TITLE = 'Associated objects';
 
-export const ASSC_OBJ_PANEL_DESRIPTION = 'Manage objects associated with this data sources.';
+export const ASSC_OBJ_PANEL_DESCRIPTION = 'Manage objects associated with this data sources.';
```

Comment:
If we have time could we scan through for all these newly added text to see if i18n are added?